src/site/shared/utils/CircuitInfoHelpers.ts

```diff
@@ -2,8 +2,8 @@ import {CircuitMetadata} from "core/models/CircuitMetadata";
 
 
 export type CircuitInfoHelpers = {
-    LoadCircuit: (getData: () => Promise<string>) => Promise<void>;
-    SaveCircuitRemote: () => Promise<boolean>;
+    LoadCircuit: (getData: () => Promise<string | unknown>) => Promise<void>;
+    SaveCircuitRemote: () => Promise<boolean | unknown>;
```

Comment:
unknown?? nonono

Reply:
This does result in some issues with something like `await helpers.LoadCircuit(() => LoadUserCircuit(auth, circuit.getId()));` (in `SideNav`) because `() => LoadUserCircuit(auth, circuit.getId())` is `LoadUserCircuit(auth: AuthState, circuitId: string): Promise<string | undefined>`